PennMobile/GSR-Booking/Views/RoomFinderSelectionPanel.swift

```diff
@@ -142,6 +142,15 @@ struct RoomFinderSelectionPanel: View {
                 self.maxTimeRequirement = 90
             }
         }
+        .alert(item: $quickBook.activeAlert) { alert in
+            Alert(title: Text(alert.title), message: Text(alert.message), primaryButton: .default(Text("OK")) {
```

Comment:
[The `Alert` struct has recently been deprecated.](https://developer.apple.com/documentation/swiftui/alert) Can we switch to the [other form](https://developer.apple.com/documentation/swiftui/view/alert(_:ispresented:presenting:actions:message:)-8584l) of `.alert` instead?